scripts/count-imports/parser.js

```diff
@@ -0,0 +1,27 @@
+const babylon7 = require('babylon7');
+
+const options = {
+  sourceType: 'module',
+  allowImportExportEverywhere: false,
+  allowReturnOutsideFunction: false,
+  ranges: false,
+  tokens: false,
+  plugins: [
+    'asyncGenerators',
+    'classProperties',
+    // "decorators",
+    'doExpressions',
+    'exportExtensions',
+    'flow',
+    'functionSent',
+    'functionBind',
+    'jsx',
+    'objectRestSpread',
+    'dynamicImport',
+    'numericSeparator',
+    'optionalChaining',
+    'optionalCatchBinding',
```

Comment:
can consider removing the followings unneeded plugins
```
decorators,
doExpressions,
functionSent,
numericSeparator,
optionalChaining
```

Reply:
Removed!